# SOME DESCRIPTIVE TITLE.
# Copyright (C) YEAR THE PACKAGE'S COPYRIGHT HOLDER
# This file is distributed under the same license as the PACKAGE package.
# FIRST AUTHOR <EMAIL@ADDRESS>, YEAR.
# 
# Translators:
# Mindaugas <opensuse.lietuviu.kalba@gmail.com>, 2019
# Moo, 2023
# 
#, fuzzy
msgid ""
msgstr ""
"Project-Id-Version: PACKAGE VERSION\n"
"Report-Msgid-Bugs-To: \n"
"POT-Creation-Date: 2023-09-28 22:49+0200\n"
"PO-Revision-Date: 2017-08-09 10:34+0000\n"
"Last-Translator: Moo, 2023\n"
"Language-Team: Lithuanian (https://app.transifex.com/calamares/teams/20061/lt/)\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"
"Language: lt\n"
"Plural-Forms: nplurals=4; plural=(n % 10 == 1 && (n % 100 > 19 || n % 100 < 11) ? 0 : (n % 10 >= 2 && n % 10 <=9) && (n % 100 > 19 || n % 100 < 11) ? 1 : n % 1 != 0 ? 2: 3);\n"

#: src/modules/bootloader/main.py:46
msgid "Install bootloader."
msgstr "Įdiegti operacinės sistemos paleidyklę."

#: src/modules/bootloader/main.py:644
msgid "Failed to install grub, no partitions defined in global storage"
msgstr ""
"Nepavyko įdiegti grub paleidyklės, visuotinėje saugykloje nėra apibrėžta "
"jokių skaidinių"

#: src/modules/bootloader/main.py:899
msgid "Bootloader installation error"
msgstr "Operacinės sistemos paleidyklės diegimo klaida"

#: src/modules/bootloader/main.py:900
msgid ""
"The bootloader could not be installed. The installation command "
"<pre>{!s}</pre> returned error code {!s}."
msgstr ""
"Nepavyko įdiegti operacinės sistemos paleidyklės. Diegimo komanda "
"<pre>{!s}</pre> grąžino klaidos kodą {!s}."

#: src/modules/displaymanager/main.py:509
msgid "Cannot write LXDM configuration file"
msgstr "Nepavyksta įrašyti LXDM konfigūracijos failą"

#: src/modules/displaymanager/main.py:510
msgid "LXDM config file {!s} does not exist"
msgstr "LXDM konfigūracijos failo {!s} nėra"

#: src/modules/displaymanager/main.py:598
msgid "Cannot write LightDM configuration file"
msgstr "Nepavyksta įrašyti LightDM konfigūracijos failą"

#: src/modules/displaymanager/main.py:599
msgid "LightDM config file {!s} does not exist"
msgstr "LightDM konfigūracijos failo {!s} nėra"

#: src/modules/displaymanager/main.py:684
msgid "Cannot configure LightDM"
msgstr "Nepavyksta konfigūruoti LightDM"

#: src/modules/displaymanager/main.py:685
msgid "No LightDM greeter installed."
msgstr "Neįdiegtas joks LightDM pasisveikinimas."

#: src/modules/displaymanager/main.py:716
msgid "Cannot write SLIM configuration file"
msgstr "Nepavyksta įrašyti SLIM konfigūracijos failą"

#: src/modules/displaymanager/main.py:717
msgid "SLIM config file {!s} does not exist"
msgstr "SLIM konfigūracijos failo {!s} nėra"

#: src/modules/displaymanager/main.py:935
msgid "No display managers selected for the displaymanager module."
msgstr "Displaymanagers moduliui nėra pasirinkta jokių ekranų tvarkytuvių."

#: src/modules/displaymanager/main.py:936
msgid ""
"The displaymanagers list is empty or undefined in both globalstorage and "
"displaymanager.conf."
msgstr ""
"Displaymanagers sąrašas yra tuščias arba neapibrėžtas tiek globalstorage, "
"tiek ir displaymanager.conf faile."

#: src/modules/displaymanager/main.py:1023
msgid "Display manager configuration was incomplete"
msgstr "Ekranų tvarkytuvės konfigūracija yra nepilna"

#: src/modules/dracut/main.py:29
msgid "Creating initramfs with dracut."
msgstr "Sukuriama initramfs naudojant dracut."

#: src/modules/dracut/main.py:63
msgid "Failed to run dracut"
msgstr "Nepavyko paleisti dracut"

#: src/modules/dracut/main.py:64
#, python-brace-format
msgid "Dracut failed to run on the target with return code: {return_code}"
msgstr ""
"Dracut nepavyko pasileisti paskirties vietoje su grįžties kodu: "
"{return_code}"

#: src/modules/dummypython/main.py:35
msgid "Dummy python job."
msgstr "Fiktyvi python užduotis."

#: src/modules/dummypython/main.py:37 src/modules/dummypython/main.py:93
#: src/modules/dummypython/main.py:94
msgid "Dummy python step {}"
msgstr "Fiktyvus python žingsnis {}"

#: src/modules/fstab/main.py:28
msgid "Writing fstab."
msgstr "Rašoma fstab."

#: src/modules/fstab/main.py:377 src/modules/fstab/main.py:383
#: src/modules/fstab/main.py:411 src/modules/initcpiocfg/main.py:256
#: src/modules/initcpiocfg/main.py:260 src/modules/initramfscfg/main.py:85
#: src/modules/initramfscfg/main.py:89 src/modules/localecfg/main.py:140
#: src/modules/mount/main.py:329 src/modules/networkcfg/main.py:105
#: src/modules/openrcdmcryptcfg/main.py:72
#: src/modules/openrcdmcryptcfg/main.py:76 src/modules/rawfs/main.py:164
msgid "Configuration Error"
msgstr "Konfigūracijos klaida"

#: src/modules/fstab/main.py:378 src/modules/initramfscfg/main.py:86
#: src/modules/mount/main.py:330 src/modules/openrcdmcryptcfg/main.py:73
#: src/modules/rawfs/main.py:165
msgid "No partitions are defined for <pre>{!s}</pre> to use."
msgstr "Nėra apibrėžta jokių skaidinių, skirtų <pre>{!s}</pre> naudojimui."

#: src/modules/fstab/main.py:384 src/modules/initramfscfg/main.py:90
#: src/modules/localecfg/main.py:141 src/modules/networkcfg/main.py:106
#: src/modules/openrcdmcryptcfg/main.py:77
msgid "No root mount point is given for <pre>{!s}</pre> to use."
msgstr ""
"Nėra nurodyta jokių šaknies prijungimo taškų, skirtų <pre>{!s}</pre> "
"naudojimui."

#: src/modules/fstab/main.py:412
msgid "No <pre>{!s}</pre> configuration is given for <pre>{!s}</pre> to use."
msgstr ""
"Nenurodyta jokia <pre>{!s}</pre> konfigūracija, kurią <pre>{!s}</pre> galėtų"
" naudoti."

#: src/modules/grubcfg/main.py:29
msgid "Configure GRUB."
msgstr "Konfigūruoti GRUB."

#: src/modules/hwclock/main.py:26
msgid "Setting hardware clock."
msgstr "Nustatomas aparatinės įrangos laikrodis."

#: src/modules/initcpiocfg/main.py:27
msgid "Configuring mkinitcpio."
msgstr "Konfigūruojama mkinitcpio."

#: src/modules/initcpiocfg/main.py:257
msgid "No partitions are defined for <pre>initcpiocfg</pre>."
msgstr "Nėra apibrėžta jokių skaidinių, skirtų <pre>initcpiocfg</pre>."

#: src/modules/initcpiocfg/main.py:261
msgid "No root mount point for <pre>initcpiocfg</pre>."
msgstr "Nėra šaknies prijungimo taško, skirto <pre>initcpiocfg</pre>."

#: src/modules/initramfscfg/main.py:32
msgid "Configuring initramfs."
msgstr "Konfigūruojama initramfs."

#: src/modules/localecfg/main.py:31
msgid "Configuring locales."
msgstr "Konfigūruojamos lokalės."

#: src/modules/mkinitfs/main.py:27
msgid "Creating initramfs with mkinitfs."
msgstr "Kuriama initramfs naudojant mkinitfs."

#: src/modules/mkinitfs/main.py:49
msgid "Failed to run mkinitfs on the target"
msgstr "Nepavyko paskirties vietoje paleisti mkinitfs"

#: src/modules/mkinitfs/main.py:50
msgid "The exit code was {}"
msgstr "Išėjimo kodas buvo {}"

#: src/modules/mount/main.py:43
msgid "Mounting partitions."
msgstr "Prijungiami skaidiniai."

#: src/modules/mount/main.py:164 src/modules/mount/main.py:200
msgid "Internal error mounting zfs datasets"
msgstr "Vidinė klaida prijungiant zfs duomenų rinkinius"

#: src/modules/mount/main.py:176
msgid "Failed to import zpool"
msgstr "Nepavyko importuoti zpool"

#: src/modules/mount/main.py:192
msgid "Failed to unlock zpool"
msgstr "Nepavyko atrakinti zpool"

#: src/modules/mount/main.py:209 src/modules/mount/main.py:214
msgid "Failed to set zfs mountpoint"
msgstr "Nepavyko nustatyti zfs prijungimo taško"

#: src/modules/mount/main.py:365
msgid "zfs mounting error"
msgstr "zfs prijungimo klaida"

#: src/modules/networkcfg/main.py:29
msgid "Saving network configuration."
msgstr "Įrašoma tinklo konfigūracija."

#: src/modules/openrcdmcryptcfg/main.py:26
msgid "Configuring OpenRC dmcrypt service."
msgstr "Konfigūruojama OpenRC dmcrypt tarnyba."

#: src/modules/packages/main.py:54 src/modules/packages/main.py:65
#: src/modules/packages/main.py:75
msgid "Install packages."
msgstr "Įdiegti paketus."

#: src/modules/packages/main.py:63
#, python-format
msgid "Processing packages (%(count)d / %(total)d)"
msgstr "Apdorojami paketai (%(count)d / %(total)d)"

#: src/modules/packages/main.py:68
#, python-format
msgid "Installing one package."
msgid_plural "Installing %(num)d packages."
msgstr[0] "Įdiegiamas %(num)d paketas."
msgstr[1] "Įdiegiami %(num)d paketai."
msgstr[2] "Įdiegiama %(num)d paketų."
msgstr[3] "Įdiegiama %(num)d paketų."

#: src/modules/packages/main.py:71
#, python-format
msgid "Removing one package."
msgid_plural "Removing %(num)d packages."
msgstr[0] "Šalinamas %(num)d paketas."
msgstr[1] "Šalinami %(num)d paketai."
msgstr[2] "Šalinama %(num)d paketų."
msgstr[3] "Šalinama %(num)d paketų."

#: src/modules/packages/main.py:740 src/modules/packages/main.py:752
#: src/modules/packages/main.py:780
msgid "Package Manager error"
msgstr "Paketų tvarkytuvės klaida"

#: src/modules/packages/main.py:741
msgid ""
"The package manager could not prepare updates. The command <pre>{!s}</pre> "
"returned error code {!s}."
msgstr ""
"Paketų tvarkytuvei nepavyko paruošti atnaujinimų. Komanda <pre>{!s}</pre> "
"grąžino klaidos kodą {!s}."

#: src/modules/packages/main.py:753
msgid ""
"The package manager could not update the system. The command <pre>{!s}</pre>"
" returned error code {!s}."
msgstr ""
"Paketų tvarkytuvei nepavyko atnaujinti sistemos. Komanda <pre>{!s}</pre> "
"grąžino klaidos kodą {!s}."

#: src/modules/packages/main.py:781
msgid ""
"The package manager could not make changes to the installed system. The "
"command <pre>{!s}</pre> returned error code {!s}."
msgstr ""
"Paketų tvarkytuvei nepavyko atlikti pakeitimų įdiegtoje sistemoje. Komanda "
"<pre>{!s}</pre> grąžino klaidos kodą {!s}."

#: src/modules/plymouthcfg/main.py:27
msgid "Configure Plymouth theme"
msgstr "Konfigūruoti Plymouth temą"

#: src/modules/rawfs/main.py:26
msgid "Installing data."
msgstr "Įdiegiami duomenys."

#: src/modules/services-openrc/main.py:29
msgid "Configure OpenRC services"
msgstr "Konfigūruoti OpenRC tarnybas"

#: src/modules/services-openrc/main.py:57
msgid "Cannot add service {name!s} to run-level {level!s}."
msgstr "Nepavyksta pridėti tarnybą {name!s} į vykdymo lygmenį {level!s}."

#: src/modules/services-openrc/main.py:59
msgid "Cannot remove service {name!s} from run-level {level!s}."
msgstr "Nepavyksta pašalinti tarnybą {name!s} iš vykdymo lygmens {level!s}."

#: src/modules/services-openrc/main.py:61
msgid ""
"Unknown service-action <code>{arg!s}</code> for service {name!s} in run-"
"level {level!s}."
msgstr ""
"Nežinomas tarnybos veiksmas <code>{arg!s}</code>, skirtas tarnybai {name!s} "
"vykdymo lygmenyje {level!s}."

#: src/modules/services-openrc/main.py:93
msgid "Cannot modify service"
msgstr "Nepavyksta modifikuoti tarnybos"

#: src/modules/services-openrc/main.py:94
msgid ""
"<code>rc-update {arg!s}</code> call in chroot returned error code {num!s}."
msgstr ""
"<code>rc-update {arg!s}</code> iškvieta, esanti chroot, grąžino klaidos kodą"
" {num!s}."

#: src/modules/services-openrc/main.py:101
msgid "Target runlevel does not exist"
msgstr "Paskirties vykdymo lygmens nėra"

#: src/modules/services-openrc/main.py:102
msgid ""
"The path for runlevel {level!s} is <code>{path!s}</code>, which does not "
"exist."
msgstr ""
"Vykdymo lygmens {level!s} kelias yra <code>{path!s}</code>, kurio savo "
"ruožtu nėra."

#: src/modules/services-openrc/main.py:110
msgid "Target service does not exist"
msgstr "Paskirties tarnybos nėra"

#: src/modules/services-openrc/main.py:111
msgid ""
"The path for service {name!s} is <code>{path!s}</code>, which does not "
"exist."
msgstr ""
"Tarnybos {name!s} kelias yra <code>{path!s}</code>, kurio savo ruožtu nėra."

#: src/modules/services-systemd/main.py:26
msgid "Configure systemd units"
msgstr "Konfigūruoti systemd įtaisus"

#: src/modules/services-systemd/main.py:64
msgid "Cannot modify unit"
msgstr "Nepavyksta modifikuoti įtaiso"

#: src/modules/services-systemd/main.py:65
msgid ""
"<code>systemctl {_action!s}</code> call in chroot returned error code "
"{_exit_code!s}."
msgstr ""
"<code>systemctl {_action!s}</code> iškvieta chroot viduje grąžino klaidos "
"kodą {_exit_code!s}."

#: src/modules/services-systemd/main.py:66
msgid "Cannot {_action!s} systemd unit <code>{_name!s}</code>."
msgstr "Nepavyksta {_action!s} systemd <code>{_name!s}</code> įtaiso."

#: src/modules/unpackfs/main.py:34
msgid "Filling up filesystems."
msgstr "Užpildomos failų sistemos."

#: src/modules/unpackfs/main.py:254
msgid "rsync failed with error code {}."
msgstr "rsync patyrė nesėkmę su klaidos kodu {}."

#: src/modules/unpackfs/main.py:299
msgid "Unpacking image {}/{}, file {}/{}"
msgstr "Išpakuojamas atvaizdis {}/{}, failas {}/{}"

#: src/modules/unpackfs/main.py:314
msgid "Starting to unpack {}"
msgstr "Pradedama išpakuoti {}"

#: src/modules/unpackfs/main.py:323 src/modules/unpackfs/main.py:467
msgid "Failed to unpack image \"{}\""
msgstr "Nepavyko išpakuoti atvaizdį „{}“"

#: src/modules/unpackfs/main.py:430
msgid "No mount point for root partition"
msgstr "Nėra prijungimo taško šaknies skaidiniui"

#: src/modules/unpackfs/main.py:431
msgid "globalstorage does not contain a \"rootMountPoint\" key."
msgstr "globalstorage viduje nėra „rootMountPoint“ rakto."

#: src/modules/unpackfs/main.py:434
msgid "Bad mount point for root partition"
msgstr "Blogas šaknies skaidinio prijungimo taškas"

#: src/modules/unpackfs/main.py:435
msgid "rootMountPoint is \"{}\", which does not exist."
msgstr "„rootMountPoint“ yra „{}“, kurio savo ruožtu tiesiog nėra."

#: src/modules/unpackfs/main.py:439 src/modules/unpackfs/main.py:455
#: src/modules/unpackfs/main.py:459 src/modules/unpackfs/main.py:465
#: src/modules/unpackfs/main.py:480
msgid "Bad unpackfs configuration"
msgstr "Bloga unpackfs konfigūracija"

#: src/modules/unpackfs/main.py:440
msgid "There is no configuration information."
msgstr "Nėra jokios informacijos apie konfigūraciją."

#: src/modules/unpackfs/main.py:456
msgid "The filesystem for \"{}\" ({}) is not supported by your current kernel"
msgstr "Jūsų branduolys nepalaiko failų sistemos, kuri skirta „{}“ ({})"

#: src/modules/unpackfs/main.py:460
msgid "The source filesystem \"{}\" does not exist"
msgstr "Šaltinio failų sistemos „{}“ nėra"

#: src/modules/unpackfs/main.py:466
msgid ""
"Failed to find unsquashfs, make sure you have the squashfs-tools package "
"installed."
msgstr ""
"Nepavyko rasti unsquashfs, įsitikinkite, kad esate įdiegę squashfs-tools "
"paketą."

#: src/modules/unpackfs/main.py:481
msgid "The destination \"{}\" in the target system is not a directory"
msgstr "Paskirties vieta „{}“, esanti paskirties sistemoje, nėra katalogas"

#: src/modules/zfshostid/main.py:27
msgid "Copying zfs generated hostid."
msgstr "Kopijuojamas zfs sugeneruotas hostid."
